tests/testthat/test-merge_sce_list.R

```diff
@@ -20,7 +28,7 @@ set.seed(1665)
 total_cells <- 24 # divisible by 3
 total_genes <- 12 # number of months intentionally.
 sce <- add_sce_data(
-  sim_sce(n_cells = total_cells, n_genes = total_genes, n_empty = 0)
+  scpcaTools:::sim_sce(n_cells = total_cells, n_genes = total_genes, n_empty = 0)
```

Comment:
The `scpcaTools:::` should not be needed within the test suite, is it? (You might need it for running the code in the console, but not for running via testing tools)

Reply:
No sorry that's just leftover from testing on my own.